content/spin/v1/dynamic-configuration.md

```diff
@@ -231,7 +220,9 @@ By default, components will not have access to any of these databases (even the
 
 ### LLM Runtime Configuration
 
-Spin provides a Large Language Model interface for interacting with LLMs for inferencing and embedding. The default host implementation is to use local CPU/GPU compute. However, the Spin runtime configuration file (runtime-config.toml) can be updated to enable Spin to use remote compute using HTTP requests.
+Spin provides a Large Language Model interface for interacting with LLMs for inferencing and embedding. The default host implementation is to use local CPU/GPU compute. However, the Spin runtime configuration file (`runtime-config.toml`) can be updated to enable Spin to use remote compute using HTTP requests.
+
+### Remote Compute Provider
```

Comment:
In other cases you've made the provider config one heading level deeper than the "area" it applies to.